### PR TITLE
Updated file for Belfast, NI recommendations

### PR DESCRIPTION
**Pull request by df-2017**
initial commit

### CODE REVIEW

**Comment by df-2017** [comment]
closes #14

File: df-2017-belfast.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Belfast Recommendations
+
+Some recommendations for visiting Belfast, NI.
+
+### Places to eat
+
```
Comment (beardofedu) [inline df-2017-belfast.md]: Where can I get a good 🍺 and some solid Corned Beef?

File: df-2017-belfast.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Belfast Recommendations
+
```
Comment (beardofedu) [inline df-2017-belfast.md]: Creating a review for @df-2017

File: df-2017-belfast.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+Some recommendations for visiting Belfast, NI.
+
+### Places to eat
+
+
+### Things to do
```
Comment (beardofedu) [inline df-2017-belfast.md]: Where should I go in Belfast?

**Pull request merged**